### ISSUE TITLE
Docker starts and dies unexpectedly

### ISSUE THREAD

**Issue by mikila85** [Deci-AI/super-gradients#1154]
### 🐛 Describe the bug

Hi,
I am trying to run your docker pulled straight from `deciai/super-gradients:latest`:
![image](https://github.com/Deci-AI/super-gradients/assets/3965957/055a2681-0cfd-4f53-8a0c-d0fb11649894)
it goes up, and writes:
![image](https://github.com/Deci-AI/super-gradients/assets/3965957/3f2ee482-9b5c-4058-8094-125a3ecbb230)
And just closes.

Windows 10, Nvidia 3070, 
any ideas what is wrong?
am I doing something wrong?

Thank you.

### Versions

Windows user it doesnt work <3

**Comment by BloodAxe** [comment]
I may be wrong, but to my knowledge, GPU-passthrough inside docker containers was never supported on Windows platform.

**Comment by mikila85** [comment]
ow :( @BloodAxe  am I wasting my time trying to run it in windows?
I wasnt able to install it directly either due to the pycocotools problem no matter what I tried..
Stay safe there in Ukraine

**Comment by mmoollllee** [comment]
Same here on a mac. Can't install super-gradients due to pycocotools & onnx "Could not build wheels" errors.
So tried it with docker, which can't be launched without CUDA? Just want to test pre-trained on CPU...

**Comment by BloodAxe** [comment]
In 3.2.1 we have updated version of pycocotools which should enable to build install SG on Win & Mac platforms.
For Windows you may need to install C++ build tools to compile pycocotools. But other than that `pip install super_gradients==3.1.3` should run smoothly.
